documentation/getting-started.adoc

```diff
@@ -53,6 +53,68 @@ can be used:
 [source]
 oc describe all
 
+=== Topic Controller
+
+Strimzi uses a component called the Topic Controller to manage topics in the Kafka cluster. The Topic Controller
+is deployed as a process running inside your Kubernetes or OpenShift cluster. To create a new Kafka topic, a ConfigMap
+with the related configuration (name, partitions, replication factor, ...) has to be created. Based on the information
+in that ConfigMap, the Topic Controller will create a corresponding Kafka topic in the cluster.
+
+Deleting a topic ConfigMap raises the deletion of the corresponding Kafka topic as well.
+
+Strimzi allows to deploy the Topic Controller through the Cluster Controller. The cluster ConfigMap contains the Topic
+Controller configuration for doing that.
+
+Furthermore, Strimzi contains example YAML files for deploying the Topic Controller not through the Cluster Controller as well.
+
+==== Deploying through the Cluster Controller
+
+To deploy the Topic Controller through the Cluster Controller, its configuration needs to be provided in the cluster
+ConfigMap in the `topic-controller-config` field as a JSON string.
+
+For more information on the JSON configuration format see <<topic_controller_json_config>>.
+
+Of course, such a deployment comes "out of box" from deploying the Cluster Controller as explained in <<Cluster Controller>>.
```

Comment:
I'm not sure what is this actually trying to say ... is it trying to say that it is enabled by default? It is not, or?

Reply:
It should say that there is no topic controller "specific" action needed for deploying it through the CC. Just configure it and deploy the CC. No YAML files needed for the TC.